css/style.css

```diff
@@ -4,7 +4,114 @@
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    margin: 0;
+    padding: 0;
+    border: 1px solid white
 }
+*{transition:all 0.3s ease-in-out;}
+*{transition:all 0.3s ease-in-out;}
+
+.container{
+    clear: both;
+    overflow: auto;
+}
+
+nav{float:right;}
+
+.logo img{
+    position: relative;
+    float: left;
```

Comment:
Avoid using `float` and `clear` rules. This is antiquated and has been replaced by flexbox. Flexbox offers a more reliable way of laying out elements, and once you get used to it it'll become very easy 🙂

Reply:
Remove the float rules and change your `container` rules to use flexbox. Do you remember how to do this? Have a look [here](https://www.freecodecamp.org/news/an-animated-guide-to-flexbox-d280cf6afc35/) if not 😉